lib/standard/collection/union_find.nit

```diff
@@ -158,6 +157,36 @@ class DisjointSet[E]
 		return true
 	end
 
+	# Returns an array of all elements in the same subset as `e`.
+	#
+	#     var s = new DisjointSet[Int]
+	#     s.add_all([1,2,3,4,5,6])
+	#     s.union(1,2)
+	#     s.union(1,3)
+	#     s.union(4,5)
+	#     assert s.subset(1).length == 3
+	#     assert s.subset(4).length == 2
+	#     assert s.subset(6).length == 1
+	fun subset(e: E): Array[E]
+	do
+		return [for f in nodes.keys
+			do if find(e) == find(f) then f]
+	end
+
+	# Returns an array of all subsets of this disjoint set
+	#
+	#    var s = new DisjointSet[Int]
+	#    s.add_all([1,2,3,4,5,6])
+	#    s.union(1,2)
+	#    s.union(1,3)
+	#    s.union(4,5)
+	#    assert s.all_subsets.length == 3
+	fun all_subsets: Array[Array[E]]
```

Comment:
Maybe I did not understand the point but what is the difference between the new method `all_subsets` and the existing method `to_partitions`? Even the examples are basically the same.